src/prefect/orion/schemas/core.py

```diff
@@ -207,16 +206,16 @@ class FlowRun(ORMBaseModel):
     state_name: Optional[str] = Field(
         default=None, description="The name of the current flow run state."
     )
-
     run_count: int = Field(
         default=0, description="The number of times the flow run was executed."
     )
-
+    restarts: int = Field(
```

Comment:
Just to echo my comment above, I can't understand why `run_count` doesn't already contain this information. How can run count increment if the flow wasn't restarted?

Reply:
Just for posterity: the run count increments already with flow retries, restarts were conceived as a separate concept--we decided to make them the same so `restarts` -> `manual_retries` which consume a retry attempt